bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -493,10 +493,29 @@ offtrials = [f"offtrial_{i+1}" for i in range(num_trials)]
 # Sensitivity plots of each injection set
 out_dir = rdir['exclusion_distances']
 _workflow.makedir(out_dir)
+bkgd_plots = []
+for inj_set in inj_sets:
+    # Retrieve the injections result File for the inj_set label
+    inj_file = inj_files.find_output_with_tag(inj_set,
+                                              fail_if_not_single_file=True)
+    tags = [offtrials[0]] + inj_file.tags
+    plot_node, output_files = \
+        _workflow.make_pygrb_plot(wflow, 'pygrb_efficiency',
+                                  out_dir, trig_file=offsource_file,
+                                  inj_file=inj_file,
+                                  bank_file=bank_file,
+                                  seg_files=seg_files, tags=tags,
+                                  plot_bkgd=True)
```

Comment:
Why is `plot_bkgd` an explicit kwarg? Why can whether or not to plot the background[?] be controlled with command line options set via the tags? (e.g. `tags=tags+['background']` and then add a [pygrb_efficiency-background] section with the relevant option?)